.github/workflows/image-build-binary.yml

```diff
@@ -0,0 +1,135 @@
+name: Docker Image (Binary)
```

Comment:
I don't have a good sense of the immediate purpose for some of these workflows. I don't doubt there's a good reason so I think these files could benefit from a little "workflow justification" at the top so that if one of us gets hit by a bus the workflow is self justifying.

Reply:
yea, some context may be helpful - this workflow is 100% required though, it creates the tagged docker images that are published to dockerhub (these are all bulilt from the binary artifacts you've asked about above). 

some better comments (as you have noted) might be a good idea